### DISCUSSION TITLE
Welcome to care_fe Discussions!

### DISCUSSION THREAD

**Discussion by dauntlessnomad** [ohcnetwork/care_fe#957] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tdTilak** [comment]
Hey Man just wanted to appreciate your work, great work!

**Comment by v1nshul** [comment]
Hello, I would like to contribute to your project, im new to open source, if you could help me with how can i get started then i will really appreciate it.
